Makefile

```diff
@@ -0,0 +1,8 @@
+FEATURES=testing
+
+install:
+	cargo install --features=${FEATURES} --path node
+
+run:
+	miden-node make-genesis --inputs-path node/genesis.toml
+	miden-node start --config node/miden-node.toml
```

Comment:
The goal is to move from Makefile to `cargo make`, could you try using that instead?

reference: https://github.com/0xPolygonMiden/miden-node/issues/251

Reply:
Changing to that

Reply:
Will do so in another PR that we will merge before this one. 

Closing this issue: #251